cmd/scollector/collectors/program.go

```diff
@@ -115,7 +116,12 @@ func (c *ProgramCollector) Init() {
 var setupExternalCommand = func(cmd *exec.Cmd) {}
 
 func (c *ProgramCollector) runProgram(dpchan chan<- *opentsdb.DataPoint) (progError error) {
-	cmd := exec.Command(c.Path)
+	var cmd *exec.Cmd
+	if runtime.GOOS == "windows" && filepath.Ext(c.Path) == ".ps1" {
+		cmd = exec.Command("powershell", "-NoProfile", "-ExecutionPolicy", "Bypass", "-File", c.Path)
```

Comment:
What do you think about adding -NoLogo and -NonInteractive here? We usually use those in our collector scripts.

I think -ExecutionPolicy Bypass can fail on systems that [set it via GPO](https://absolute-sharepoint.com/2014/03/change-powershell-machinepolicy-execution-policy-windows-server-2012r2.html). Might be best to have this as optional either via an ENV var **SCOLLECTOR_PS_POLICY=byapss** or part of the scollector.toml config file. Or we can wait for someone to report an issue and address it at that time.